rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -768,4 +768,56 @@ namespace rsx
 			}
 		}
 	}
+
+	template <typename T, typename bitmask_type = u32>
+	class atomic_bitmask_t
```

Comment:
missing XOR op?

Reply:
Didn't have a use for it, but it can be added.